apps/web/src/containers/App/index.tsx

```diff
@@ -79,20 +72,10 @@ export const App = () => {
     }
   }, []);
 
-  useEffect(() => {
-    initPlausible();
-    trackPageview();
-  }, [trackPageview]);
-
   useEffect(() => {
     initApplication();
```

Comment:
What does this call do? Do we need to remove it as well?

Reply:
It calls the above initApplication that sets up onboard so we still need it